EC-84: Restructure eZ Commerce doc, part 2 (#2)
ezsystems/commerce-documentation: docs/guide/product_description/erp_integration.md

|Process|Details|Advantage|
|--- |--- |--- |

> |Login for B2B customer|The shop updates customer data from the ERP in realtime|The shop is using the customer data from the ERP. Always up-to-date.</br>The customer is always using the same data regardless of the channel they are using.|

Reviewer: ```suggestion
|Login for B2B customer|The shop updates customer data from the ERP in a real-time|The shop is using the customer data from the ERP. Always up-to-date.</br>The customer is always using the same data regardless of the channel they are using.|
```

|Activate B2B account|The shop can validate an account request using the customer number and other parts of an invoice (e.g. invoice number).|Existing customers can create a shop account and get access within minutes which minimizes the internal work for the shop owner.|

> |Realtime stock|The shop can request stock information in realtime.|The stock is updated in real time.|

Reviewer: ```suggestion
|Realtime stock|The shop can request stock information in a real-time.|The stock is updated in real-time.|
```

|Realtime prices|The prices used in the shop are coming directly from the ERP. The shop uses the business logic of the ERP.|Prices are always up to date. Complex price rules are used without duplicating the logic.|
|Orders|Orders are directly transferred to the ERP.|The customer gets feedback (e.g. order number) in a few seconds.|
|Documents|The shop requests orders, delivery notes, invoices and credit memos.|The customer can get and print documents for all channels (online, offline orders).</br>Less risks since the data is not duplicated to the shop server.|
|Products|Some ERP systems provides methods to import product data from the ERP.|Automatic product synchronization|

## Fallback scenario

eZ Commerce supports fallback scenarios for the most important processes in case the connection to the ERP is not available:

- The latest customer data is cached after login and used if the ERP is not available after login.
- A fallback price engine is used in case the ERP is not available. The customer is informed that the prices and stock is not up to date.

> - An order will be stored in the shop. A job will take care that the order is transmitted to the ERP when it is available again.

Reviewer: ```suggestion
- An order will be stored in the shop. A job will transmit the order to the ERP when it is available again.
```

> ## Realtime stock information during checkout and in product detail page

Reviewer: ```suggestion
## Real-time stock information during checkout and in product detail page
```

The performance monitor allows to find and identify bottlenecks between eZ Commerce and the ERP system. 

> It enables analyzing the communication for a given time period. The view can be filtered by a time period and allows to identify the message which are performing good or not good. 

Reviewer: ```suggestion
It enables analyzing the communication for a given time period. The view can be filtered by a time period and allows to identify how messages are performing. 
```
